### PR TITLE
docs(README): rm `beta` install reference

### PR DESCRIPTION
**Pull request by backjo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 <h2 align="center">Install</h2>
 
 ```bash
-npm i -D uglifyjs-webpack-plugin@beta
+npm i -D uglifyjs-webpack-plugin@latest
```
Comment (michael-ciniawsky) [inline README.md]: Please just remove the appendix altogether :)

```js
npm i -D uglifyjs-webpack-plugin
```

Comment (backjo) [reply README.md]: done!

**Comment by kzc** [comment]
This sentence in the README is longer correct for `uglifyjs-webpack-plugin@1.x`:

> ℹ️ webpack contains the same plugin under `webpack.optimize.UglifyJsPlugin`. The documentation is valid apart from the installation instructions

Although it may be correct for a future `webpack` release.

**Comment by michael-ciniawsky** [comment]
@kzc Fixed by #153 please review if anything still unclear or needs rewording :)

**Pull request merged**